kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeStagingDataTab.js

```diff
@@ -65,38 +68,73 @@ define([
 
         render: function() {
             return this.getUserInfo()
-            .then(userInfo => {
-                var $mainElem = $('<div>')
-                    .css({
-                        'height': '604px',
-                        'padding': '5px',
-                        'overflow-y': 'auto'
+                .then(userInfo => {
+                    const $mainElem = $('<div>')
```

Comment:
could you give this a class and put the style info into a css file?

Reply:
Good idea. I mostly went through and fiddled with layout and minor linting in that file, but didn't adjust CSS. I'll put that in a class.

Reply:
Ok, done.